presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -800,6 +800,32 @@ Examples:
 
     CALL iceberg.system.expire_snapshots(schema => 'schema_name', table_name => 'table_name', snapshot_ids => ARRAY[10001, 10002]);
 
+Remove Orphan Files
+^^^^^^^^^^^^^^^^^^^
+
+Used to remove files which are not referenced in any metadata files of an Iceberg table and can thus be considered "orphaned".
```

Comment:
```suggestion
Use to remove files which are not referenced in any metadata files of an Iceberg table.
```

Reply:
Thanks for the suggestion, fixed! Please take a look when available.